.github/workflows/cxx_interop.yml

```diff
@@ -0,0 +1,38 @@
+name: Cxx interop
+
+on:
+  workflow_call:
+    inputs:
+        linux_5_8_enabled:
+            type: boolean
+            description: "Boolean to enable the Linux 5.8 Swift version matrix job. Defaults to true."
+            default: true
+        linux_5_9_enabled:
+            type: boolean
+            description: "Boolean to enable the Linux 5.9 Swift version matrix job. Defaults to true."
+            default: true
+        linux_5_10_enabled:
+            type: boolean
+            description: "Boolean to enable the Linux 5.10 Swift version matrix job. Defaults to true."
+            default: true
+        linux_nightly_6_0_enabled:
+            type: boolean
+            description: "Boolean to enable the Linux nightly 6.0 Swift version matrix job. Defaults to true."
+            default: true
+        linux_nightly_main_enabled:
+            type: boolean
+            description: "Boolean to enable the Linux nightly main Swift version matrix job. Defaults to true."
+            default: true
+ 
+jobs:
+    cxx-interop:
+        name: Cxx interop
+        uses: ./.github/workflows/pull_request_swift_matrix.yml
+        with:
+          name: "Cxx interop"
+          matrix_linux_command: "apt-get update -y -q && apt-get install -y -q jq && ./scripts/check-cxx-interop-compatibility.sh"
```

Comment:
Does this rely on the adopter having this script in their repo? If so, should we make this pipeline opt-in rather than opt-out?